From 5d4024d909b75fddee8c03a6434458a9b2f721db Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 25 Nov 2018 18:56:38 +0200
Subject: [PATCH] version: move ifdeffery outside of the function

Do not pollute the function code/logic by the build target ifdeffery.
Instead provide a stub for the complementary case.

Change-Id: Ia1bbad8dc9103590919ae2ec50dec21a9556ff19
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/preboot.c b/preboot.c
index 40eb772..550927c 100644
--- a/preboot.c
+++ b/preboot.c
@@ -456,9 +456,9 @@
 	hl_preboot_puts("PCIe link is UP!\n");
 }
 
+#ifndef TARGET_IS_TESTER
 static void hl_preboot_version(void)
 {
-#ifndef TARGET_IS_TESTER
 	hl_preboot_puts("\n");
 	hl_preboot_puts_raw("Preboot ");
 	hl_preboot_puts_raw(PREBOOT_VER);
@@ -467,8 +467,10 @@
 	hl_preboot_puts_raw(" - ");
 	hl_preboot_puts_raw(PREBOOT_TIME);
 	hl_preboot_puts(")\n");
-#endif
 }
+#else	/* TARGET_IS_TESTER */
+static inline void hl_preboot_version(void) {}
+#endif	/* TARGET_IS_TESTER */
 
 void hl_preboot_main(void)
 {
